defaults/main.yml

```diff
@@ -30,12 +33,14 @@ rsnapshot_config_intervals:
       value: 3
 rsnapshot_config_verbose: 2
 rsnapshot_config_loglevel: 3
-rsnapshot_config_logfile: /var/log/rsnapshot.log
-rsnapshot_config_lockfile: /var/run/rsnapshot.pid
+rsnapshot_config_logfile: /var/log/rsnapshot/rsnapshot.log
+rsnapshot_config_lockfile: /var/run/rsnapshot/rsnapshot.pid
```

Comment:
I just saw that and I'm not sure to understand why these subdirectories are needed for the not_root feature?

Reply:
Because, a non-root can't write in /var/log and /var/run. We need to create a subdir with owner the user who will run rsnapshot.